coral-service/README.md

```diff
@@ -0,0 +1,42 @@
+# Coral-as-a-Service
```

Comment:
How about we add this in the main README?

Reply:
Do you mean adding Coral Service's entire README in Coral's README, or do you mean simply adding "coral-service" under "Modules" in [Coral's README](https://github.com/linkedin/coral/blob/master/README.md)?

Reply:
The former.